### PR TITLE
Background information (non-normative)

### PR DESCRIPTION
**Pull request by BurtHarris**
Also minor edits to address warnings from markdownlint.

### CODE REVIEW

**Comment by BurtHarris** [comment]
While this isn't by any means complete, comments on GitHub are welcome.

**Comment by cabo** [comment]
Hi Burt,

this PR is hard to review as there are so many gratuitous source formatting changes mixed in with the new content.  As a general rule, we try to refrain from source formatting changes in version controlled sources, unless there is a strong reason for those changes -- in that case, the formatting changes would be a separate commit (but they will still obscure history).

**Comment by BurtHarris** [comment]
@cabo, The fist commit contains the substantitve changes.   Others were to resolve tool warnings, etc.  You can review the first in isolation.

**Comment by BurtHarris** [comment]
I'm still working out the warnings cnd checks...

File: draft-ietf-cbor-7049bis.md [hunk 1]
```diff
@@ -37,15 +37,15 @@ normative:
     target: http://www.ecma-international.org/publications/files/ecma-st/ECMA-262.pdf
     title: ECMAScript Language Specification 5.1 Edition
     author:
-    - org: European Computer Manufacturers Association
+      - org: European Computer Manufacturers Association
```
Comment (BurtHarris) [inline draft-ietf-cbor-7049bis.md]: markdownlint recommends YAML indentation that makes clear this list is subordinate to the author key.

**Comment by BurtHarris** [comment]
@cabo the above comments reflect the basis for the sort of formatting changes I suspect you found distracting.   If you really want, when I'm done addressing the remaining failures, I can reorganize the commits within the PR to separate these out better, but I think they are all justified.

File: draft-ietf-cbor-7049bis.md [hunk 2]
```diff
@@ -2306,6 +2326,7 @@ well_formed_indefinite(mt, breakable) {
   return 0;                     // no break out
 }
 ~~~~
+
```
[the remark this answers is not in the document]
Comment (cabo) [reply draft-ietf-cbor-7049bis.md]: I haven't checked whether this actively breaks the build but there is no point in a blank line between a figure and its IAL (title, anchor).  It seems to me that markdownlint is creating more problems than it solves.

[the remark this answers is not in the document]
Comment (BurtHarris) [reply draft-ietf-cbor-7049bis.md]: P.P.S  As a general rule, I think extra whitespace when switching languages is a good idea in a polyglot community.   

```
{: #pseudo title='Pseudocode for Well-Formedness Check'}
```

is Greek to me.

[the remark this answers is not in the document]
Comment (BurtHarris) [reply draft-ietf-cbor-7049bis.md]: See github/linguist project

**Comment by BurtHarris** [comment]
@cabo, I found your IET92 kramdown-rfc [tutorial](https://www.ietf.org/about/participate/tutorials/process/writing-rfcs-and-internet-drafts-markdown-and-bit-yaml/), looks like it will help me out.

**Comment by BurtHarris** [comment]
@cabo on page 36 of your tutorial slides, it says:

> **Ending blocks**
> 
> Blocks are usually separated by a blank line.
> 
> Line breaks don't matter, unless line ends in
> 
> - two spaces NOT RECCOMENDED
> 
> - \\ (kramdown featue) 
> 

And the later discussion of IALs.   I gather from that the extra whitespace line between code and IAL shouldn't matter, but what's the deal with trailing whitespace?   Does this not work like YAML and ignore invisible whitespace, like trailing whitespace?

**Comment by BurtHarris** [comment]
P.S. the secret meaning behind my photo substitute icon is shaking up the angle brackets.

**Comment by BurtHarris** [comment]
@cabo I'm having problems getting the reversion I wanted to make to stick.   I'm going to close this and resubmit in a day or so from another branch, but I'd still be interested in your responses above.

**Pull request closed without merging**